control-plane/config/sink/100-kafka-sink.yaml

```diff
@@ -65,8 +65,11 @@ spec:
                   type: integer
                   format: int32
                 bootstrapServers:
-                  description: 'A comma separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster.'
-                  type: string
+                  description: 'A list of host/port pairs to use for establishing the initial connection to the Kafka cluster.'
+                  type: array
+                  minLength: 1
+                  items:
+                    type: string
```

Comment:
I get this:

```
The CustomResourceDefinition "kafkasinks.eventing.knative.dev" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[bootstrapServers].items: Required value: must be specified
```

I am on k8s 1.18.x

Reply:
It should be fixed now, please refresh your branch.